### PR TITLE
Adding the pom.xml and modifying .gitignore

### PR DESCRIPTION
**Pull request by asniii**
Also added the launchClient file

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.github</groupId>
```
Comment (Shailendre) [inline pom.xml]: Hi,
we should change the groupid to info.lazycompiler. This is the purchased domain that this project should belong.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <artifactId>FXReportEngine</artifactId>
```
Comment (Shailendre) [inline pom.xml]: all artifacts should be lowercase.
Thanks

**Review by Shailendre — CHANGES_REQUESTED** [review]
Please take actions as per comments

File: pom.xml (hunk 1 above)
Comment (asniii) [reply pom.xml]: ok

File: pom.xml (hunk 2 above)
Comment (asniii) [reply pom.xml]: ok

**Review by Shailendre — APPROVED** [review]
ok.

**Pull request merged**